feature-libs/cart/base/core/services/cart-config.service.spec.ts

```diff
@@ -20,7 +20,12 @@ describe('CartConfigService', () => {
 
   describe('isSelectiveCartService', () => {
     it('should return true if selectiveCart enabled is set to true', () => {
-      mockCartConfig.cart = { selectiveCart: { enabled: true } };
+      mockCartConfig.cart = {
+        selectiveCart: { enabled: true },
+        addToCartFeedback: {
+          feedback: ADDED_TO_CART_FEEDBACK.MODAL,
+        },
```

Comment:
I don't think the add to cart feedback should be mocked in this spec.  It doesn't seem related to the test.

Reply:
I added this since the test was failing, due to a lack of default behavior when no feedback config is provided.